corehq/apps/users/models_sql.py

```diff
@@ -84,6 +84,22 @@ class Meta:
             models.Index(fields=("couch_id",)),
         )
 
+    @classmethod
+    @transaction.atomic
+    def create(cls, domain, name, permissions=None, assignable_by=None, **kwargs):
+        from corehq.apps.users.models import Permissions
+        role = SQLUserRole.objects.create(domain=domain, name=name, **kwargs)
```

Comment:
Should this be in a `disable_sync_to_couch` context manager? Won't this call `save` on the sql object and cause a sync to couch, which wouldn't get rolled back if there's an issue in the middle of this function?

Reply:
d2ef58404e0